src/_includes/templates/forms/accessibility-feedback-form.html

```diff
@@ -0,0 +1,201 @@
+                   type="text">
+        </div>
+
+        <div class="form-l_col form-l_col-1-2">
+            <label class="form-label-header"
+                   for="accessibility-form_email-address">
+                Email address
+            </label>
+            <input class="input__long"
+                   id="accessibility-form_email-address"
+                   type="text">
+        </div>
+
+        <div class="form-l_col form-l_col-1-2">
+            <label class="form-label-header"
+                   for="accessibility-form_contact-time">
+                Best time to reach you
+            </label>
+            <div class="custom-select">
+                <select class="custom-select_select"
+                        data-placeholder="Select Time"
+                        id="accessibility-form_contact-time">
+                      <option value="">Select time</option>
+                      {# TODO: Add additional times. #}
+                      <option>8:00 a.m</option>
+                 </select>
+                 <span class="custom-select_icon
+                                 cf-icon
+                                 cf-icon-down"></span>
+            </div>
+        </div>
+
+        <div class="form-l_col form-l_col-1">
+            <label class="form-label-header"
+                   for="accessibility-form_complaint-desc">
+                Nature of the accessibility problem*
+            </label>
+            <textarea class="input__long"
+                      id="accessibility-form_complaint-desc"
+                      required>
+            </textarea>
+        </div>
+
+        <div class="form-l_col form-l_col-1">
+            <label class="form-label-header"
+                   for="accessibility-form_address-request-material">
+                Web address (URL) of requested material
+            </label>
+            <input class="input__long"
+                   id="accessibility-form_address-request-material"
+                   type="text">
+        </div>
+
+        <div class="form-l_col form-l_col-1">
+            <label class="form-label-header"
+                   for="accessibility-form_preferred-format">
+                Preferred format*
+            </label>
+            <div class="custom-select">
+                <select class="custom-select_select"
+                        data-placeholder="Select Time"
+                        id="accessibility-form_preferred-format"
+                        required>
+                      <option value="">Select format</option>
+                      {# TODO: Add additional formats. #}
+                      <option>PDF</option>
+                 </select>
+                 <span class="custom-select_icon
+                                 cf-icon
+                                 cf-icon-down"></span>
+            </div>
+        </div>
+
+        <div class="form-l_col form-l_col-1 form-actions form-actions__right-on-med ">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/4847ab5ff92bdd1120188fdf493aa63a9fa7f193